fullstack-cert/js-projects/space-mission-roster/user-stories.md

```diff
@@ -0,0 +1,11 @@
+1. You should create a `crew` array that stores astronaut objects with `id`, `name`, `role`, `evaEligible`, and `priority`.
```

Comment:
this file can be removed since you are working on a workshop and not a lab.
we only add user stories for labs. not workshops. 

the user stories in the case were more for you to build your prototype. but they don't need to be added here. 👍🏾